docs/crd.md

```diff
@@ -0,0 +1,87 @@
+# Restoring Deleted CRs
+
+When a Rook CR is deleted, the Rook operator will respond to the deletion event to attempt to clean up the cluster resources. If any data is still present in the cluster, Rook will refuse to delete the CR to ensure data is not lost. The operator will refuse to remove the finalizer on the CR until the underlying data is deleted.
+
+While the underlying Ceph data and daemons continue to be available, the CRs will be stuck indefinitely in a Deleting state in which the operator will not continue to ensure cluster health. Upgrades will be blocked, further updates to the CRs are prevented, and so on. Since Kubernetes does not allow undeleting resources, the command below will allow repairing the CRs without even necessarily suffering cluster downtime.
+
+## Restore Command
+
+- `<CRD>`: the CRD type that is to be restored, such as CephCluster, CephFilesystem, CephBlockPool and so on.
+- `[CRName]`: is the name of the specific CR which you want to restore since there can be multiple instances under the same CRD. For example, if there are multiple CephFilesystems stuck in deleting state, a specific filesystem can be restored: `restore-deleted cephfilesystem filesystem-2`.
+
+```bash
+kubectl rook-ceph restore-deleted <CRD> [CRName]
+
+Info: Detecting which resources to restore for crd "cephcluster"
+Info: Restoring CR my-cluster
+Warning: The resource my-cluster was found deleted. Do you want to restore it? yes | no
+
+Info: skipped prompt since ROOK_PLUGIN_SKIP_PROMPTS=true
+Info: Scaling down the operator to 0
+Info: Backing up kubernetes and crd resources
+---
```

Comment:
Why are there so many separators? `---`

Reply:
Ah, I would expect `...` to represent removing some lines

Reply:
updated